templates/contactList.html

```diff
@@ -13,4 +13,7 @@
 			<h2>{{ctrl.t.emptySearch}}</h2>
 		</div>
 	</div>
+	<div ng-if="filtered.length>0" class="contacts-list-count">
+		{{ filtered.length }} {{ filtered.length==1 ? ctrl.t.oneContact : ctrl.t.multipleContacts }}
```

Comment:
@DeepDiver1975 do you remember how/where we fixed this before?
@juliushaertl Please take another look. We would be happy to merge this PR after this is resolved!